### PR TITLE
HPCC-13540 Fixed ulimit regression from HPCC-12780

### PR DESCRIPTION
**Pull request by Michael-Gardner**
Signed-off-by: Michael Gardner Michael.Gardner@lexisnexis.com

### CODE REVIEW

**Comment by Michael-Gardner** [comment]
@jakesmith Modifications for the regression from HPCC-12780.

Reverted the changes to add_conf_settings.sh and start_slaves.

File: initfiles/componentfiles/thor/start_slaves [hunk 1]
```diff
@@ -68,8 +68,10 @@ done
 trap "rm -f $lckfile" exit
 echo $$ > $lckfile
 
-
-
+ulimit -c unlimited > /dev/null 2>&1
+[[ $? -ne 0 ]] && echo "`date` Failed to set ulimit for core file size"
+ulimit -n 32768 > /dev/null 2>&1
```
Comment (jakesmith) [inline initfiles/componentfiles/thor/start_slaves]: I know it wan't before, but should ulimit -S be used ?

**Comment by jakesmith** [comment]
Looks good to me.
@richardkchapman - ready to merge.

**Comment by AttilaVamos** [comment]
Automated Smoketest Build: success
HPCC Start: OK
HPCC Stop: OK

**Pull request merged**